.github/workflows/doxygen-flow.yml

```diff
@@ -0,0 +1,49 @@
+name: Update Documentation
+
+on:
+  release:
+    types: [published]
+
+jobs:
+ updatdocs:
+    name: Update Doxygen Documentation
+    runs-on: macos-latest
```

Comment:
Not urgent again, but macOS runners are not fully supported in Github Actions. They are still a preview feature and might eventually get deprecated. Since nothing in this workflow needs macOS specifically, it might be better to port this over to Ubuntu, replacing `brew` commands with `apt-get`.
Can be a separate follow-up commit.